src/test/scala/uk/gov/hmrc/perftests/itsass/IncomeTaxSubmissionSimulation.scala

```diff
@@ -146,32 +147,71 @@ class IncomeTaxSubmissionSimulation extends PerformanceTestRunner {
   setup("individual-employment", "Individual Single Employment Journey") withRequests(
     getLoginPage,
     postIndividualLoginPage("AA133742A"),
-    getStartPage,
-    getOverviewPage,
-    getEmploymentSummaryPage,
-    getEmploymentDetailsPage("00000000-0000-1000-8000-000000000000"),
-    getEmploymentBenefitsPage("00000000-0000-1000-8000-000000000000"),
-    getEmploymentExpensesPage
+    getStartPage(taxYear),
+    getOverviewPage(taxYear),
+    getEmploymentSummaryPage(taxYear),
+    getEmploymentDetailsPage(taxYear, "00000000-0000-1000-8000-000000000000"),
+    getEmploymentBenefitsPage(taxYear,"00000000-0000-1000-8000-000000000000"),
+    getEmploymentExpensesPage(taxYear)
   )
 
   setup("agent-employment", "Agent Multiple Employments Journey") withRequests(
     getLoginPage,
     postAgentLoginPage("BB444444A"),
     getInsertAdditionalParametersEndPoint("BB444444A"),
-    getStartPage,
-    getOverviewPage,
-    getEmploymentSummaryPage,
-    getEmployerDetailsAndBenefitsPage("00000000-5555-0000-0000-000000000001"),
-    getEmploymentDetailsPage("00000000-5555-0000-0000-000000000001"),
-    getEmploymentBenefitsPage("00000000-5555-0000-0000-000000000001"),
-    getEmploymentExpensesPage
+    getStartPage(taxYear),
+    getOverviewPage(taxYear),
+    getEmploymentSummaryPage(taxYear),
+    getEmployerDetailsAndBenefitsPage(taxYear, employmentId="00000000-5555-0000-0000-000000000001"),
+    getEmploymentDetailsPage(taxYear, employmentId="00000000-5555-0000-0000-000000000001"),
+    getEmploymentBenefitsPage(taxYear, employmentId="00000000-5555-0000-0000-000000000001"),
+    getEmploymentExpensesPage(taxYear)
+  )
+
+  setup("individual-add-employment", "Individual Add Employment -No Employments") withRequests(
```

Comment:
space between the "-" and "No", sorry to be picky

Reply:
done